src/mavedb/routers/experiments.py

```diff
@@ -192,6 +195,26 @@ async def create_experiment(
     for publication in publication_identifiers:
         setattr(publication, "primary", publication.identifier in primary_identifiers)
 
+    # Controlled keywords currently is allowed none. Will be changed in the future.
```

Comment:
If we plan to change this, let's add an item to the backlog and reference it here.